py3status/modules/bluetooth.py

```diff
@@ -121,7 +118,7 @@ def post_config_hook(self):
     def bluetooth(self):
         try:
             devices = get_connected_devices()
-        except dbus.DBusException:
+        except Exception:
```

Comment:
I can't trigger this. Switching to `pydbus` might make `try/except` + `STRING_NOT_STARTED` useless cruft.

Reply:
Same, I'll just get rid of this entirely then

Reply:
Bump.